### PR TITLE
Document static files in Express

### LINKED ISSUE #113: Should document `static_assets`
The website should provide information about serving up static files using the `static_assets` parameter of `App()` (Core) and `app_opts()` (Express).

### LINKED ISSUE #117: Shiny Express can now serve static files with v0.8.0
https://shiny.posit.co/py/docs/ui-customize.html#serve-local-files

https://github.com/posit-dev/py-shiny-site/blob/429ace2e0f031998cd9a55bc16b06b4d8a9c82d6/docs/ui-customize.qmd#L226-L228

Reported on Discord: https://discord.com/channels/1109483223987277844/1112869981735104553/1214516951658004580

### PR DESCRIPTION
**Pull request by wch**
Closes #117, closes #113.

### CODE REVIEW

**Comment by wch** [comment]
I tried using `.shiny-mode-tabset` and  `.panel-pills` for the Express/Core switcher. Neither of them is ideal, because there's no boundary line to distinguish what's inside of the tab vs what's outside. But I think the pills are a little less confusing.


https://github.com/posit-dev/py-shiny-site/assets/86978/313ada4a-52e7-46f9-b32a-537d264445eb


https://github.com/posit-dev/py-shiny-site/assets/86978/c930e941-9a14-4131-a159-8c055e7e42cc

**Review by gadenbuie — APPROVED** [review]
Looks great, I just have a few small comments and suggestions

**Comment by gadenbuie** [comment]
> But I think the pills are a little less confusing.

Of the two, I definitely like the pills better!

File: docs/ui-customize.qmd [hunk 1]
```diff
@@ -195,7 +195,63 @@ That said, if you want your app to work properly offline, you'll want to serve t
...
+#| viewerHeight: 150
+
+from shiny.express import ui
+
+ui.tags.link(href="my-styles.css", rel="stylesheet")
+
+with ui.card():
+    ui.card_header("Card header")
+    "Card body"
+## file: www/my-styles.css
```
[the remark this answers is not in the document]
Comment (wch) [reply docs/ui-customize.qmd:219]: I agree! I actually prefer it that way, but I was using the same style as other examples in this file.

One thing that's actually good about having a newline there, and at the end of the next file, is that then there will be a trailing newline for those files in the editor.

**Pull request merged**